packages/dds/tree/src/shared-tree/treeAlpha.ts

```diff
@@ -124,6 +128,8 @@ identifier.create = (branch: TreeBranch): string => {
 
 Object.freeze(identifier);
 
+export type ParentObject = TreeView<ImplicitFieldSchema> | TreeNode;
```

Comment:
You want any branch to be usable as this type. Using `TreeView<ImplicitFieldSchema> ` has two distinct issues:

1. TreeView is invariant over its schema, not covariant, so it will be impossible to actually use this for a view with any schema that is not exactly `ImplicitFieldSchema`. Well, it might work as the write type for the root ends up being never which might happen to accidently work if TS checks is structurally, but its not supposed to work. (see https://en.wikipedia.org/wiki/Type_variance )
2. It won't work for branches which are not the main view.

You want to use `TreeBranch` instead.